### ISSUE TITLE
Traceback will be generated when processing an onnx model.

### ISSUE THREAD

**Issue by whyb** [tsingmicro-toolchain/OnnxSlim#14]
This model is modified based on the [DiffIR](https://github.com/Zj-BinXia/DiffIR) open source model.
command:
```bash
onnxslim input.onnx output.onnx --input_shapes input:1,3,-1,-1
```
I will get the following result as shown in the picture:
![圖片](https://github.com/tsingmicro-toolchain/OnnxSlim/assets/2672931/47c5f549-fe6c-4413-a9e4-032ae46509da)

Environment:
* OnnxSlim Version: 0.1.22
* Onnx Version: 1.14.1
* OnnxRuntime Version: 1.17.1
* Python Version: Python 3.10.9
* OS: Windows 10 22H2

**Comment by inisis** [comment]
duplicate of https://github.com/tsingmicro-toolchain/OnnxSlim/issues/9, I will release a new version soon

**Comment by whyb** [comment]
Solved👍🏻, works now in new version 0.1.23, Thanks
